src/components/Person/Person.jsx

```diff
@@ -1 +1,17 @@
-// export const Person = ({ person }) => ();
+export const Person = ({ person }) => (
+  <section className="Person">
+    <h2 className="Person__name">{`My name is ${person.name}`}</h2>
+
+    {person.age !== undefined && (
+      <p className="Person__age">I am 37</p>
+    )}
+    <p className="Person__partner">
+      {!person.isMarried && `I am not married`}
+      {person.isMarried
+        && (person.sex === `m`
+          ? `${person.partnerName} is my wife`
+          : `${person.partnerName} is my husband`)
```

Comment:
You can create a separate variable to avoid code duplication

`const partner = person.sex === 'm' ? 'wife' : 'husband'`